UKHO.FmEssFssMock.API/UKHO.FmEssFssMock.API/Services/FileShareService.cs

```diff
@@ -167,15 +167,18 @@ public BatchStatusResponse GetBatchStatus(string batchId, string homeDirectoryPa
 
             if (FileHelper.ValidateFilePath(batchFolderPath) && FileHelper.CheckFolderExists(batchFolderPath))
             {
-                batchStatusResponse.BatchId = batchId;
-                batchStatusResponse.Status = "Committed";
+                if (File.Exists(Path.Combine(batchFolderPath, "CommitInProgress.txt")))
```

Comment:
This can be simplified as follows:

            if (FileHelper.ValidateFilePath(batchFolderPath) && FileHelper.CheckFolderExists(batchFolderPath))
            {
                batchStatusResponse.BatchId = batchId;
                batchStatusResponse.Status = GetBatchStatus(batchFolderPath);
            }


With a new helper function as follows:

        private string GetBatchStatus(string path) => File.Exists(Path.Combine(path, "CommitInProgress.txt")) ? "CommitInProgress" : "Committed";

Code duplication also removed.

Reply:
Replaced